### DISCUSSION TITLE
p/dd0e9cb99dbb4be1badccb58f3354294/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#27] [Announcements]
# p/dd0e9cb99dbb4be1badccb58f3354294/

刚过完五一，连休了 9 天之后又开始上班了。上周回到了广州父母那边，不过非常不凑巧的是妈妈回老家处理一些事情，所以只有爸爸在那边。回去这几天基本上只有周六是一整天都在家里，其他时候都是出去见朋友和之前的同事。     时间 事情    04-29 周六 飞回广州，结果飞机晚点 2 个钟，凌晨到家   04-30 周日 白天去朋友家里玩   05-01 周一 住了一晚，第二天下午回家   05-02

https://blog.rxliuli.com/p/dd0e9cb99dbb4be1badccb58f3354294/

**Comment by miaopass-future** [comment]
难怪突然用不了了呢，我也想去众筹买一个了，感觉有点依赖上了。感谢博主免费体验到现在 O(∩_∩)O

**Comment by Tshiwu** [comment]
太强了
